### PR TITLE
Hide dashboard elements if only one canvas view

### PR DESCRIPTION
**Pull request by bsekachev**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx [hunk 1]
```diff
@@ -72,7 +72,10 @@ const fitLayout = (type: DimensionType, layoutConfig: ItemLayout[]): ItemLayout[
     const relatedViews = layoutConfig
         .filter((item: ItemLayout) => item.viewType === ViewType.RELATED_IMAGE);
     const relatedViewsCols = relatedViews.length > 6 ? 2 : 1;
-    const height = Math.floor(config.CANVAS_WORKSPACE_ROWS / (relatedViews.length / relatedViewsCols));
+    let height = Math.floor(config.CANVAS_WORKSPACE_ROWS / (relatedViews.length / relatedViewsCols));
+    if (height > 4) {
+        height = 4;
+    }
```
Comment (klakhov) [inline cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx:78]: Why do we limit the height?

Comment (bsekachev) [reply cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx:78]: According to maximum default height for a context image

Comment (klakhov) [reply cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx:78]: Maybe we should use `config.DEFAULT_CONTEXT_IMAGE_HEIGHT`

Comment (bsekachev) [reply cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx:78]: Maybe..

**Review by klakhov — APPROVED** [review]

**Pull request merged**